website/source/docs/commands/kv/delete.html.markdown.erb

```diff
@@ -30,10 +34,6 @@ Usage: `consul kv delete [options] KEY_OR_PREFIX`
 
 * `-recurse` - Recursively delete all keys with the path. The default value is
   false.
-  
-#### Enterprise Options
-
-<%= partial "docs/commands/http_api_namespace_options" %>
```

Comment:
Should we include namespaces in one of the KV examples?

Reply:
We could add enterprise specific examples. Although I am not sure showing somehow `-ns=foo` would be more helpful than just presenting the option as being there.

Reply:
I will track the extra examples for a future beta.